src/Drivers/TwilioDriver.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace EscolaLms\TemplatesSms\Drivers;
+
+use Aloha\Twilio\Twilio;
+use Aloha\Twilio\TwilioInterface;
+use EscolaLms\TemplatesSms\Enums\ConfigEnum;
+use EscolaLms\TemplatesSms\Drivers\Contracts\SmsDriver;
+
+class TwilioDriver implements SmsDriver
+{
+    private TwilioInterface $twilio;
+
+    public function __construct()
```

Comment:
Or you could have $sid, $token, $from as constructor parameters here, and pass them from config in manager (as Manager has ConfigRepository injected, so $this->config->get(KEY) can be used)